app/services/hyrax/autopopulation/redis_storage.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Hyrax
+  module Autopopulation
+    class RedisStorage
+      attr_accessor :doi_list, :orcid_list, :hyrax_orcid_settings
+
+      def initialize(doi_list: [], orcid_list: [], hyrax_orcid_settings: {})
+        @doi_list = doi_list
+        @orcid_list = orcid_list
+        @hyrax_orcid_settings = hyrax_orcid_settings
+      end
+
+      # A wrapper to Redis sadd for saving a SeT & Redis mapped_hmset for saving hash
```

Comment:
For those of us that are not super familiar with the internal redis commands, it could be great to capitalize those commands in the comments, so its clear what they are. 
```ruby
# A wrapper to Redis SADD for saving a SET & Redis mapped_hmset for saving hash
```